user/useradm.go

```diff
@@ -30,19 +34,19 @@ var (
 	ErrUserNotInitial = errors.New("user database not empty")
 )
 
-type UserAdmApp interface {
+type App interface {
```

Comment:
it's more if a generic discussion point:

Should interface be defined in package using UserAdm type or user package that provides implementation? 

Currently its in User package which means that we always depend on importing user package even if all we need is interface definition - lets say we mock it in test or we use alternative implementation that origins from different package.